configure.ac

```diff
@@ -338,6 +338,7 @@ m4_foreach([cflag],[
         [-mno-cet],
         [-Qunused-arguments],
         [-Wno-bool-operation],
+        [-Wno-missing-braces],
```

Comment:
Yes, related. We are using aggretate initialization for static data. Aggregate initialization is recursive so when you have a VlWide inside a VlUnpacked, you could initialize it with nested braces:
```
const VlUnpacked<VlWide<3>,2> = {
  { a, b, c },
  { d, e, f }
};
```
The nested braces are completely optional according to the C++ spec (https://en.cppreference.com/w/cpp/language/aggregate_initialization), and so long as you enumerate all values in the list, they do not matter. Clang however sometimes (but not always...) warns if you omitt the nested braces. What is worse, is you might need different number of braces based on the implementation of `std::array` used in `VlUnpacked`, so what is the right number of braces might change if you swap in a different standard library implementation. Based on that, I turned the warning off and went with no nestd braces.

I could attempt to remove this, but that would need removing the std::array from VlUnpacked and replacing it with a native array (like in VlWide), so we don't depent on the stdlib implementation. We seem to export a const_iterator from VlUnpacked, but I don't see any way of actually getting an instance of that, so this might be OK. VlUnpacked probably would be cleaner without the std::array, if it's not necessary, what do you think?

Reply:
Ok, I have removed the need for disabling that warning by changing `VlUnpacked`. Note you need double braces to please all compilers, the first one is for the struct, the second one is for the m_storage within:
```
extern const VlUnpacked<VlWide<3>/*79:0*/, 256> Vt_case_auto1__ConstPool__TABLE_fa2d00f3_0 = {{
    {{ // VlWide 0
        0x20202020, 0x6c652020, 0x00006964
    }},
    // ... Omitted
}};
```